release.ps1

```diff
@@ -50,30 +50,57 @@ ce octopus package upload --package="./packages/${PROJECT_NAME}.${env:VERSION}.z
   --overwrite-mode="overwrite" `
   --no-prompt
 
+# Invoke the REST API to get all spaces
+$spacesResponse = Invoke-RestMethod -Uri "${env:OCTOPUS_CLI_SERVER}/api/spaces" -Headers @{ "X-Octopus-ApiKey" = ${env:OCTOPUS_CLI_API_KEY} }  -Method Get
```

Comment:
Any reason why you use this rest api endpoint instead of the github action that showed yesterday?

Reply:
Before creating a release if we  dont  set build information it is not getting captured in the event and hence was not working.

So i have added build information post package upload and before release so that i gets  captured

Reply:
release when we update build information after release is created 
`https://octopus.usxpress.io/api/Spaces-2/releases/Releases-58038`

release when we update build information before release is created 
`https://octopus.usxpress.io/api/Spaces-2/releases/Releases-58043`

Reply:
What if we run before the release creation?